.circleci/config.yml

```diff
@@ -97,12 +99,60 @@ jobs:
             coverage run --source=super_gradients -m unittest tests/deci_core_unit_test_suite_runner.py
             coverage report
             coverage html  # open htmlcov/index.html in a browser
-
       - store_artifacts:
           path: htmlcov
 
       - store_artifacts:
           path: ~/sg_logs
+      
+  build_and_publish_beta:
+    description : "building the temp image and pushing to dev ecr"
+    parameters:
+       repo_name:
+         type: string
+         default: deci/super-gradients
+       version:
+          type: string
+          default: ${CIRCLE_BRANCH}
+    docker:
+       - image: cimg/base:stable-20.04
+    resource_class: small
+    steps:
+     - checkout
+     - setup_remote_docker:
+          version: 20.10.7
+          docker_layer_caching: true
+     - deci-common/container_image_build:
+           dockerfile: scripts/Dockerfile
+           repository_name: << parameters.repo_name >>
+           image_tag: "${CIRCLE_BRANCH}"
```

Comment:
same

Reply:
also, you set it as a parameter above, why not use it?

Reply:
also, if this happens only on master, all the docker images will be tagged with master